torch/nativert/common/Conv.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <cstdint>
+#include <optional>
+#include <string_view>
+
+#include <c10/macros/Export.h>
+
+namespace torch::nativert {
+
+template <typename T>
+std::optional<T> tryTo(std::string_view symbol) = delete;
+
+/*
+ * Convert a string to an integer. prefixes like "0x" or trailing whitespaces
+ * are not supported. Similayly, integer string with trailing characters like
+ * "123abc" will be rejected either.
+ */
+template <>
+TORCH_API std::optional<int64_t> tryTo<int64_t>(std::string_view symbol);
+
+/*
+ * Convert a string to a double. prefixes like "0x" or trailing whitespaces
```

Comment:
why do we assume all consumers need a full 64 bits of precision?

Reply:
Originally this function was used to parse textual format FX graph from c++ (in nativert), where we always expected double precision.